### PR TITLE
Add console install resources, update examples for streams 2.8

### PR DESCRIPTION
**Pull request by MikeEdgar**
* Removes native Kube install files to deploy an individual console
* Updates console Kafka example to use KRaft mode
* Add OLM and non-OLM resources to deploy console operator
* Add example Console CR to connect to the Kafka example

### CODE REVIEW

**Comment by MikeEdgar** [comment]
cc: @jankalinic

File: examples/console/README.md [hunk 1]
```diff
@@ -2,6 +2,5 @@
 
 This folder contains example resources that may be used to deploy the Streams for Apache Kafka Console
 
-* [Prometheus](./resources/prometheus): OpenShift resources that may be used to deploy a Prometheus operator along with a Prometheus instance and associated configuration. These resources are configured to scrape the metrics from OpenShift and Kafka instances.
 * [Kafka](./resources/kafka): Deploys an example Kafka cluster, metrics export configuration, and a KafkaUser used by the Streams for Apache Kafka Console to connect to the cluster.
-* [Console](./resources/console): Resources to deploy the Streams for Apache Kafka Console, its required service account and security bindings, and networking configuration to access the user interface.
+* [Console](./resources/console): Streams for Apache Kafka Console custom example resource for use with the Kafka example resource.
```
Comment (PaulRMellor) [inline examples/console/README.md:6]:
```suggestion
* [Console](./resources/console): Streams for Apache Kafka Console custom resource for use with the example Kafka custom resource.
```

**Review by PaulRMellor — APPROVED** [review]
Looks good. Thanks Mike.
One minor suggestion

**Review by jankalinic — APPROVED** [review]
Thanks for the PR!

**Pull request merged**